addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/handler/OnkyoHandler.java

```diff
@@ -816,4 +821,20 @@ public PercentType getVolume() throws IOException {
     public void setVolume(PercentType volume) throws IOException {
         handleVolumeSet(EiscpCommand.Zone.ZONE1, volumeLevelZone1, downScaleVolume(volume));
     }
+
+    private String guessMimeTypeFromData(byte[] data) {
+        String mimeType = RawType.DEFAULT_MIME_TYPE;
```

Comment:
Maybe you should just use https://www.eclipse.org/smarthome/documentation/javadoc/org/eclipse/smarthome/io/net/http/HttpUtil.html#guessContentTypeFromData-byte:A-

Reply:
Done